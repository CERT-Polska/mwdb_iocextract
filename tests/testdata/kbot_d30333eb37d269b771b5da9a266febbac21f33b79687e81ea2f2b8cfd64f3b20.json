{
    "botcommunity": "dobre_zbiory",
    "domains": [
        {
            "cnc": "stocks-sale.org"
        },
        {
            "cnc": "Casualtoken.com"
        },
        {
            "cnc": "stcok.info"
        },
        {
            "cnc": "Simmodule.com"
        },
        {
            "cnc": "zepsearch.org"
        },
        {
            "cnc": "MMOscript.com"
        },
        {
            "cnc": "Casualunit.com"
        },
        {
            "cnc": "lightuna.org"
        },
        {
            "cnc": "farmarco.co"
        },
        {
            "cnc": "Casualmodule.com"
        },
        {
            "cnc": "Orangeadier.com"
        },
        {
            "cnc": "Atomitz.com"
        },
        {
            "cnc": "make-light.net"
        },
        {
            "cnc": "Battletoken.com"
        },
        {
            "cnc": "Grayadier.com"
        },
        {
            "cnc": "Atommbo.com"
        },
        {
            "cnc": "Blueovator.com"
        },
        {
            "cnc": "zeps-quick.com"
        },
        {
            "cnc": "york-stock.info"
        },
        {
            "cnc": "lgiht.com"
        },
        {
            "cnc": "Atomivee.com"
        },
        {
            "cnc": "zeps-extra.com"
        },
        {
            "cnc": "hyper-zeps.org"
        },
        {
            "cnc": "sotck.net"
        },
        {
            "cnc": "shed-light.org"
        },
        {
            "cnc": "Adventuretoken.com"
        },
        {
            "cnc": "zepset.org"
        },
        {
            "cnc": "Atomiyo.com"
        },
        {
            "cnc": "zeps-java.org"
        },
        {
            "cnc": "their-stock.net"
        },
        {
            "cnc": "zepsoft.net"
        },
        {
            "cnc": "Graysetter.com"
        },
        {
            "cnc": "zeps-extra.info"
        },
        {
            "cnc": "Picozzle.com"
        },
        {
            "cnc": "this-light.org"
        },
        {
            "cnc": "Orangeoneer.com"
        },
        {
            "cnc": "Competebit.com"
        },
        {
            "cnc": "Competebase.com"
        },
        {
            "cnc": "light-sky.org"
        },
        {
            "cnc": "Strategytoken.com"
        },
        {
            "cnc": "Atomijo.com"
        },
        {
            "cnc": "Blackovator.com"
        },
        {
            "cnc": "the-zeps.org"
        },
        {
            "cnc": "this-light.info"
        },
        {
            "cnc": "york-stock.net"
        },
        {
            "cnc": "make-light.info"
        },
        {
            "cnc": "lgiht.net"
        },
        {
            "cnc": "MMOset.com"
        },
        {
            "cnc": "Strategytag.com"
        },
        {
            "cnc": "zesp.info"
        },
        {
            "cnc": "Consolestate.com"
        },
        {
            "cnc": "Consolescript.com"
        },
        {
            "cnc": "Atomnix.com"
        },
        {
            "cnc": "Consoleload.com"
        },
        {
            "cnc": "the-zeps.net"
        },
        {
            "cnc": "Pintdo.com"
        },
        {
            "cnc": "Consolemodule.com"
        },
        {
            "cnc": "stokc.info"
        },
        {
            "cnc": "zeps-now.org"
        },
        {
            "cnc": "Blueoneer.com"
        },
        {
            "cnc": "Atominix.com"
        },
        {
            "cnc": "make-light.org"
        },
        {
            "cnc": "Blackoneer.com"
        },
        {
            "cnc": "zeps-post.org"
        },
        {
            "cnc": "zeps-mart.net"
        },
        {
            "cnc": "Orangesetter.com"
        },
        {
            "cnc": "low-light.info"
        },
        {
            "cnc": "dim-light.org"
        },
        {
            "cnc": "Atomzio.com"
        },
        {
            "cnc": "Micrombo.com"
        },
        {
            "cnc": "Minivee.com"
        },
        {
            "cnc": "Battleload.com"
        },
        {
            "cnc": "light-tuna.com"
        },
        {
            "cnc": "same-light.info"
        },
        {
            "cnc": "nasdaq-stock.info"
        },
        {
            "cnc": "some-light.org"
        },
        {
            "cnc": "london-stock.com"
        },
        {
            "cnc": "Brownonaut.com"
        },
        {
            "cnc": "some-light.net"
        },
        {
            "cnc": "411-zeps.net"
        },
        {
            "cnc": "Strategystate.com"
        },
        {
            "cnc": "low-light.net"
        },
        {
            "cnc": "Atomizu.com"
        },
        {
            "cnc": "Grayonaut.com"
        },
        {
            "cnc": "zeps-card.org"
        },
        {
            "cnc": "Blackadier.com"
        },
        {
            "cnc": "london-stock.net"
        },
        {
            "cnc": "Atomido.com"
        },
        {
            "cnc": "Atomideo.com"
        },
        {
            "cnc": "Competescript.com"
        },
        {
            "cnc": "no-light.info"
        },
        {
            "cnc": "Competemodule.com"
        },
        {
            "cnc": "same-light.net"
        },
        {
            "cnc": "Competeset.com"
        },
        {
            "cnc": "zepsoft.org"
        },
        {
            "cnc": "Orangechief.com"
        },
        {
            "cnc": "Consoletoken.com"
        },
        {
            "cnc": "Brownchief.com"
        },
        {
            "cnc": "lightuna.info"
        },
        {
            "cnc": "Greenadier.com"
        },
        {
            "cnc": "Competeserver.com"
        },
        {
            "cnc": "lightuna.com"
        },
        {
            "cnc": "Atomizzle.com"
        },
        {
            "cnc": "Picotz.com"
        },
        {
            "cnc": "Atomtz.com"
        },
        {
            "cnc": "zepset.info"
        },
        {
            "cnc": "411-zeps.com"
        },
        {
            "cnc": "hyper-zeps.net"
        },
        {
            "cnc": "some-light.info"
        },
        {
            "cnc": "stocks-sale.info"
        },
        {
            "cnc": "Battlemodule.com"
        },
        {
            "cnc": "zeps-peak.com"
        },
        {
            "cnc": "Consoletag.com"
        },
        {
            "cnc": "Pinkoneer.com"
        },
        {
            "cnc": "Greenonaut.com"
        },
        {
            "cnc": "zepsoft.info"
        },
        {
            "cnc": "zeps-soft.info"
        },
        {
            "cnc": "york-stock.org"
        },
        {
            "cnc": "lgiht.info"
        },
        {
            "cnc": "MMOunit.com"
        },
        {
            "cnc": "Picombo.com"
        },
        {
            "cnc": "lightuna.net"
        },
        {
            "cnc": "lihgt.net"
        },
        {
            "cnc": "hyper-zeps.com"
        },
        {
            "cnc": "Atomndo.com"
        },
        {
            "cnc": "zeps-quick.org"
        },
        {
            "cnc": "Casualload.com"
        },
        {
            "cnc": "Atomzzle.com"
        },
        {
            "cnc": "stock-swap.net"
        },
        {
            "cnc": "light-tuna.net"
        },
        {
            "cnc": "Casualmap.com"
        },
        {
            "cnc": "Consoleserv.com"
        },
        {
            "cnc": "zesp.org"
        },
        {
            "cnc": "london-stock.info"
        },
        {
            "cnc": "Pintjo.com"
        },
        {
            "cnc": "sotck.info"
        },
        {
            "cnc": "Atomindo.com"
        },
        {
            "cnc": "sotck.org"
        },
        {
            "cnc": "Blueadier.com"
        },
        {
            "cnc": "their-stock.org"
        },
        {
            "cnc": "Brownmajor.com"
        },
        {
            "cnc": "Orangepilot.com"
        },
        {
            "cnc": "Casualsystem.com"
        },
        {
            "cnc": "light-tuna.org"
        },
        {
            "cnc": "Atomiba.com"
        },
        {
            "cnc": "stock-quotes.info"
        },
        {
            "cnc": "low-light.org"
        },
        {
            "cnc": "their-stock.com"
        },
        {
            "cnc": "Strategyload.com"
        },
        {
            "cnc": "lightrucks.info"
        },
        {
            "cnc": "Picojo.com"
        },
        {
            "cnc": "light-upon.info"
        },
        {
            "cnc": "Competestate.com"
        },
        {
            "cnc": "Graychief.com"
        },
        {
            "cnc": "Picodeo.com"
        }
    ],
    "failperiod": 60,
    "public_key": [
        "173661001596899057453986775517437275771483106203792192777136000232222570667424738113447795228137310260526445273151523163646826634358717537814776712142246968203668353690798998255133594406456662793459710470829029284739727735639826160812820383811523600725745341256796416382156720883705233348571575375724378791355",
        65537
    ],
    "serverpub": "0375D444671E5EA35D9542CEECF4596EB409F15BF99966B9DDA825132B46D87A",
    "taskperiod": 60,
    "type": "kbot",
    "version": 16777985
}