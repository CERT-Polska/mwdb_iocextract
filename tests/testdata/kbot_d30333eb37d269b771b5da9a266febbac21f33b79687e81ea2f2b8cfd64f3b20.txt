Key other:0375D444671E5EA35D9542CEECF4596EB409F15BF99966B9DDA825132B46D87A
NetLoc stocks-sale.org
NetLoc Casualtoken.com
NetLoc stcok.info
NetLoc Simmodule.com
NetLoc zepsearch.org
NetLoc MMOscript.com
NetLoc Casualunit.com
NetLoc lightuna.org
NetLoc farmarco.co
NetLoc Casualmodule.com
NetLoc Orangeadier.com
NetLoc Atomitz.com
NetLoc make-light.net
NetLoc Battletoken.com
NetLoc Grayadier.com
NetLoc Atommbo.com
NetLoc Blueovator.com
NetLoc zeps-quick.com
NetLoc york-stock.info
NetLoc lgiht.com
NetLoc Atomivee.com
NetLoc zeps-extra.com
NetLoc hyper-zeps.org
NetLoc sotck.net
NetLoc shed-light.org
NetLoc Adventuretoken.com
NetLoc zepset.org
NetLoc Atomiyo.com
NetLoc zeps-java.org
NetLoc their-stock.net
NetLoc zepsoft.net
NetLoc Graysetter.com
NetLoc zeps-extra.info
NetLoc Picozzle.com
NetLoc this-light.org
NetLoc Orangeoneer.com
NetLoc Competebit.com
NetLoc Competebase.com
NetLoc light-sky.org
NetLoc Strategytoken.com
NetLoc Atomijo.com
NetLoc Blackovator.com
NetLoc the-zeps.org
NetLoc this-light.info
NetLoc york-stock.net
NetLoc make-light.info
NetLoc lgiht.net
NetLoc MMOset.com
NetLoc Strategytag.com
NetLoc zesp.info
NetLoc Consolestate.com
NetLoc Consolescript.com
NetLoc Atomnix.com
NetLoc Consoleload.com
NetLoc the-zeps.net
NetLoc Pintdo.com
NetLoc Consolemodule.com
NetLoc stokc.info
NetLoc zeps-now.org
NetLoc Blueoneer.com
NetLoc Atominix.com
NetLoc make-light.org
NetLoc Blackoneer.com
NetLoc zeps-post.org
NetLoc zeps-mart.net
NetLoc Orangesetter.com
NetLoc low-light.info
NetLoc dim-light.org
NetLoc Atomzio.com
NetLoc Micrombo.com
NetLoc Minivee.com
NetLoc Battleload.com
NetLoc light-tuna.com
NetLoc same-light.info
NetLoc nasdaq-stock.info
NetLoc some-light.org
NetLoc london-stock.com
NetLoc Brownonaut.com
NetLoc some-light.net
NetLoc 411-zeps.net
NetLoc Strategystate.com
NetLoc low-light.net
NetLoc Atomizu.com
NetLoc Grayonaut.com
NetLoc zeps-card.org
NetLoc Blackadier.com
NetLoc london-stock.net
NetLoc Atomido.com
NetLoc Atomideo.com
NetLoc Competescript.com
NetLoc no-light.info
NetLoc Competemodule.com
NetLoc same-light.net
NetLoc Competeset.com
NetLoc zepsoft.org
NetLoc Orangechief.com
NetLoc Consoletoken.com
NetLoc Brownchief.com
NetLoc lightuna.info
NetLoc Greenadier.com
NetLoc Competeserver.com
NetLoc lightuna.com
NetLoc Atomizzle.com
NetLoc Picotz.com
NetLoc Atomtz.com
NetLoc zepset.info
NetLoc 411-zeps.com
NetLoc hyper-zeps.net
NetLoc some-light.info
NetLoc stocks-sale.info
NetLoc Battlemodule.com
NetLoc zeps-peak.com
NetLoc Consoletag.com
NetLoc Pinkoneer.com
NetLoc Greenonaut.com
NetLoc zepsoft.info
NetLoc zeps-soft.info
NetLoc york-stock.org
NetLoc lgiht.info
NetLoc MMOunit.com
NetLoc Picombo.com
NetLoc lightuna.net
NetLoc lihgt.net
NetLoc hyper-zeps.com
NetLoc Atomndo.com
NetLoc zeps-quick.org
NetLoc Casualload.com
NetLoc Atomzzle.com
NetLoc stock-swap.net
NetLoc light-tuna.net
NetLoc Casualmap.com
NetLoc Consoleserv.com
NetLoc zesp.org
NetLoc london-stock.info
NetLoc Pintjo.com
NetLoc sotck.info
NetLoc Atomindo.com
NetLoc sotck.org
NetLoc Blueadier.com
NetLoc their-stock.org
NetLoc Brownmajor.com
NetLoc Orangepilot.com
NetLoc Casualsystem.com
NetLoc light-tuna.org
NetLoc Atomiba.com
NetLoc stock-quotes.info
NetLoc low-light.org
NetLoc their-stock.com
NetLoc Strategyload.com
NetLoc lightrucks.info
NetLoc Picojo.com
NetLoc light-upon.info
NetLoc Competestate.com
NetLoc Graychief.com
NetLoc Picodeo.com
CampaignId: dobre_zbiory
