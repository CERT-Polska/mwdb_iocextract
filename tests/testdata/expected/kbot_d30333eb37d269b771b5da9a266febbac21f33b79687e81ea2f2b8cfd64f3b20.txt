Key other:0375D444671E5EA35D9542CEECF4596EB409F15BF99966B9DDA825132B46D87A
NetLoc stocks-sale.org/
NetLoc casualtoken.com/
NetLoc stcok.info/
NetLoc simmodule.com/
NetLoc zepsearch.org/
NetLoc mmoscript.com/
NetLoc casualunit.com/
NetLoc lightuna.org/
NetLoc farmarco.co/
NetLoc casualmodule.com/
NetLoc orangeadier.com/
NetLoc atomitz.com/
NetLoc make-light.net/
NetLoc battletoken.com/
NetLoc grayadier.com/
NetLoc atommbo.com/
NetLoc blueovator.com/
NetLoc zeps-quick.com/
NetLoc york-stock.info/
NetLoc lgiht.com/
NetLoc atomivee.com/
NetLoc zeps-extra.com/
NetLoc hyper-zeps.org/
NetLoc sotck.net/
NetLoc shed-light.org/
NetLoc adventuretoken.com/
NetLoc zepset.org/
NetLoc atomiyo.com/
NetLoc zeps-java.org/
NetLoc their-stock.net/
NetLoc zepsoft.net/
NetLoc graysetter.com/
NetLoc zeps-extra.info/
NetLoc picozzle.com/
NetLoc this-light.org/
NetLoc orangeoneer.com/
NetLoc competebit.com/
NetLoc competebase.com/
NetLoc light-sky.org/
NetLoc strategytoken.com/
NetLoc atomijo.com/
NetLoc blackovator.com/
NetLoc the-zeps.org/
NetLoc this-light.info/
NetLoc york-stock.net/
NetLoc make-light.info/
NetLoc lgiht.net/
NetLoc mmoset.com/
NetLoc strategytag.com/
NetLoc zesp.info/
NetLoc consolestate.com/
NetLoc consolescript.com/
NetLoc atomnix.com/
NetLoc consoleload.com/
NetLoc the-zeps.net/
NetLoc pintdo.com/
NetLoc consolemodule.com/
NetLoc stokc.info/
NetLoc zeps-now.org/
NetLoc blueoneer.com/
NetLoc atominix.com/
NetLoc make-light.org/
NetLoc blackoneer.com/
NetLoc zeps-post.org/
NetLoc zeps-mart.net/
NetLoc orangesetter.com/
NetLoc low-light.info/
NetLoc dim-light.org/
NetLoc atomzio.com/
NetLoc micrombo.com/
NetLoc minivee.com/
NetLoc battleload.com/
NetLoc light-tuna.com/
NetLoc same-light.info/
NetLoc nasdaq-stock.info/
NetLoc some-light.org/
NetLoc london-stock.com/
NetLoc brownonaut.com/
NetLoc some-light.net/
NetLoc 411-zeps.net/
NetLoc strategystate.com/
NetLoc low-light.net/
NetLoc atomizu.com/
NetLoc grayonaut.com/
NetLoc zeps-card.org/
NetLoc blackadier.com/
NetLoc london-stock.net/
NetLoc atomido.com/
NetLoc atomideo.com/
NetLoc competescript.com/
NetLoc no-light.info/
NetLoc competemodule.com/
NetLoc same-light.net/
NetLoc competeset.com/
NetLoc zepsoft.org/
NetLoc orangechief.com/
NetLoc consoletoken.com/
NetLoc brownchief.com/
NetLoc lightuna.info/
NetLoc greenadier.com/
NetLoc competeserver.com/
NetLoc lightuna.com/
NetLoc atomizzle.com/
NetLoc picotz.com/
NetLoc atomtz.com/
NetLoc zepset.info/
NetLoc 411-zeps.com/
NetLoc hyper-zeps.net/
NetLoc some-light.info/
NetLoc stocks-sale.info/
NetLoc battlemodule.com/
NetLoc zeps-peak.com/
NetLoc consoletag.com/
NetLoc pinkoneer.com/
NetLoc greenonaut.com/
NetLoc zepsoft.info/
NetLoc zeps-soft.info/
NetLoc york-stock.org/
NetLoc lgiht.info/
NetLoc mmounit.com/
NetLoc picombo.com/
NetLoc lightuna.net/
NetLoc lihgt.net/
NetLoc hyper-zeps.com/
NetLoc atomndo.com/
NetLoc zeps-quick.org/
NetLoc casualload.com/
NetLoc atomzzle.com/
NetLoc stock-swap.net/
NetLoc light-tuna.net/
NetLoc casualmap.com/
NetLoc consoleserv.com/
NetLoc zesp.org/
NetLoc london-stock.info/
NetLoc pintjo.com/
NetLoc sotck.info/
NetLoc atomindo.com/
NetLoc sotck.org/
NetLoc blueadier.com/
NetLoc their-stock.org/
NetLoc brownmajor.com/
NetLoc orangepilot.com/
NetLoc casualsystem.com/
NetLoc light-tuna.org/
NetLoc atomiba.com/
NetLoc stock-quotes.info/
NetLoc low-light.org/
NetLoc their-stock.com/
NetLoc strategyload.com/
NetLoc lightrucks.info/
NetLoc picojo.com/
NetLoc light-upon.info/
NetLoc competestate.com/
NetLoc graychief.com/
NetLoc picodeo.com/
CampaignId: dobre_zbiory
